site/src/components/Footer/index.js

```diff
@@ -208,7 +208,7 @@ const Footer = () => {
         </div>
       </div>
       <div className="container flex copyright">
-        <div className="text">&copy; 2022-<script>document.write(new Date().getFullYear())</script> The Meshery Authors</div>
+        <div className="text">&copy; 2023-<script>document.write(new Date().getFullYear())</script> The Meshery Authors</div>
```

Comment:
@kaushik-rishi This won't work, it's hardcoded, Please make it dynamic, hint - see the script next to it.

Reply:
// @Savio629

Reply:
As the year was initially hardcoded, I didn't looked properly to the script.
@kaushik-rishi Please make the required changes.
If you face any issue try to ping me...

Reply:
@Chadha93 can you check the updated pr ?